.github/workflows/test.yaml

```diff
@@ -87,7 +94,23 @@ jobs:
         with:
           tool: cargo-make
 
+      # Code coverage is generated only for amd64 because cargo-llvm-cov
+      # does not currently support Windows on arm64.
+      # See issue: https://github.com/taiki-e/cargo-llvm-cov/issues/436
+      - name: Run Cargo Test with Coverage (workspace)
+        if: matrix.runner.name == 'windows-2025'
+        run: cargo +${{ matrix.rust_toolchain }} llvm-cov --workspace --codecov --output-path target/codecov.info --locked --profile ${{ matrix.cargo_profile }} --all-features
+
+      - name: Upload Coverage to Codecov
+        if: matrix.runner.name == 'windows-2025'
+        uses: codecov/codecov-action@v5
+        with:
+          files: target/codecov.info
+          token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
Does this actually work for our fork based workflow? 

from https://github.com/codecov/codecov-action:
<img width="1264" height="202" alt="Image" src="https://github.com/user-attachments/assets/ca0a445d-9f29-45b4-b7c7-1beacb092a2a" />

Reply:
You can get your own token for your fork and Codecov will record coverage separately for it. Here's an example of where it recorded coverage for my fork of WDR: https://app.codecov.io/gh/gurry/windows-drivers-rs/tree/add-code-coverage

Reply:
May want to mention needing to do this in `contributing.md`

Reply:
I have changed the workflow to not attempt upload to codecov in forks. So this should not be a problem any more.